packages/story-editor/src/utils/getMediaBaseColor.js

```diff
@@ -19,104 +19,60 @@
  */
 import { getTimeTracker, trackError } from '@web-stories-wp/tracking';
 import { getHexFromSolidArray } from '@web-stories-wp/patterns';
+import { preloadImage } from '@web-stories-wp/media';
 
-const STYLES = {
-  boxSizing: 'border-box',
-  visibility: 'hidden',
-  position: 'fixed',
-  contain: 'layout paint',
-  top: '-9999px',
-  left: '-9999px',
-  zIndex: -1,
-};
-
-const BASE_COLOR_NODE = '__WEB_STORIES_BASE_COLOR__';
-const IMG_NODE = '__WEB_STORIES_IMG_NODE';
-
-export function getImgNodeId(elementId) {
-  if (elementId === undefined) {
-    return '__web-stories-base-color';
-  }
-  return `__web-stories-bg-${elementId}`;
-}
+/**
+ * Extracts the base color from an image element.
+ *
+ * @param {Image} img Image.
+ * @return {Promise<string>} Hex color.
+ */
+async function extractColorFromImage(img) {
+  const { default: ColorThief } = await import(
```

Comment:
Is it a good idea to load a module with `await`? Why not load it async? It should also handle failing loading, too.

Reply:
Async/await is just syntactic sugar really

Reply:
I personally just find this more readable.